examples/config files - basic/1 user-sync-config.yml

```diff
@@ -53,47 +54,48 @@ adobe_users:
   # (required) connectors
   # The connectors section specifies how to connect User Sync to Adobe.
   connectors:
+
     # (required) umapi
     # umapi stands for User Management API, and is the protocol
     # used to read and write users on the Adobe side.  The value
     # of the umapi setting is the absolute or relative pathname
     # of a file that contains your UMAPI endpoint and credential
-    # information. (See the documentation for details.)  If you
+    # information file. (See the documentation for details.)  If you
```

Comment:
Whoops.  You don't want this extra use of file, see line 62 "of a file that contains ..."